frontend/src/pages/AccountReceive.vue

```diff
@@ -178,7 +180,11 @@ function useScan() {
     const overrides = { startBlock: startBlockLocal.value, endBlock: endBlockLocal.value };
     let allAnnouncements: AnnouncementDetail[] = [];
     try {
-      allAnnouncements = await umbra.value.fetchAllAnnouncements(overrides);
+      allAnnouncements = await umbra.value.fetchSomeAnnouncements(
+        signer.value as JsonRpcSigner,
```

Comment:
Rather than cast it might be better to use `!` to indicate it won't be undefined.

Reply:
And same with the cast below

Reply:
To be even more strict you wrap this call in an if statement and handle the situation where both these variables are undefined.

Reply:
Addressed here: https://github.com/ScopeLift/umbra-protocol/pull/549/commits/d4c80ebaaeaf08f903008c081b31bd74a2a28a10